src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -800,16 +800,16 @@ private String getInvalidInfoForGoogleId(String value) {
         Assumption.assertTrue("Non-null value expected", value != null);
         Assumption.assertTrue("\""+value+"\""+  "is not expected to be a gmail address.", 
                 !value.toLowerCase().endsWith("@gmail.com"));
-        String sanitizedValue = Sanitizer.sanitizeForHtml(value);
         
         if (value.isEmpty()) {
-            return String.format(GOOGLE_ID_ERROR_MESSAGE, value, REASON_EMPTY);
+            return GOOGLE_ID_ERROR_MESSAGE;
```

Comment:
This portion seems a little dubious, we're losing the ```REASON_EMPTY``` portion which seems to be valuable information to the users and if those are stripped and every return statement of this whole flow statement seem to return the same string why are we still using a flow statement